feature/preference/src/main/java/io/github/droidkaigi/confsched2020/preference/ui/viewmodel/PreferenceViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.droidkaigi.confsched2020.preference.ui.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.distinctUntilChanged
+import io.github.droidkaigi.confsched2020.ext.combine
+import javax.inject.Inject
+
+class PreferenceViewModel @Inject constructor(): ViewModel() {
+
+    data class UiModel(
+        val isNightMode: Boolean
+    ) {
+        companion object {
+            val EMPTY = UiModel(false)
+        }
+    }
+
+    private val _nightModeLiveData: MutableLiveData<Boolean> = MutableLiveData(false)
+    private val nightModeLiveData
+        get() = _nightModeLiveData.distinctUntilChanged()
```

Comment:
`combine` do `distinctUntilChanged()`. So you don't have to create 2 properties 👍